### DISCUSSION TITLE
glitches.exe

### DISCUSSION THREAD

**Discussion by ItsEnderium** [CluckCluckChicken/Magnifier#110] [General]
### **i feel like something is wrong here**
**i just can't put my finger on what it is**
![Screenshot 2021-06-24 23 07 07](https://user-images.githubusercontent.com/82263560/123338576-fd58a380-d540-11eb-807b-08497de21fdb.png)

**Comment by davwat** [comment]
eauf https://magnifier.potatophant.net/users/-InsanityGames-
